django_cloud_deploy/django_cloud_deploy.py

```diff
@@ -50,6 +55,11 @@ def main():
                      'django_cloud_deploy, on Google Kubernetes Engine.'))
     update_parser.set_defaults(func=_update)
     update.add_arguments(update_parser)
+    deploy_parser = subparsers.add_parser(
+        'deploy',
+        description=('Deploys an existing Django project.'))
```

Comment:
I think that `deploy` might not be the best name since this changes the user's code. `migrate` isn't a great name either since Django users with think about database migrations. Maybe "cloudify"?